core/domain/story_domain_test.py

```diff
@@ -453,3 +453,13 @@ def test_is_manager(self):
         self.assertTrue(story_rights.is_manager(self.USER_ID))
         self.assertTrue(story_rights.is_manager(self.USER_ID_1))
         self.assertFalse(story_rights.is_manager('fakeuser'))
+
+    def test_story_rights_change_defaults(self):
```

Comment:
Consider using a separate test class for StoryRightsChange -- seems like an antipattern to lump everything together.

Also, not sure what the test title means, or what this test is actually testing. If you want to test to_dict(), then construct a story_rights object without invoking to_dict beforehand. Call to_dict() on that object and make sure it matches what you expect.

To test `__init__()`, write a separate test that constructs a story rights object by passing in certain args, then validates the fields to check that the values are as expected.